eclipse-collections-code-generator/src/main/resources/api/primitiveIterable.stg

```diff
@@ -127,6 +127,140 @@ public interface <name>Iterable extends PrimitiveIterable
         }
     }
 
+    /**
+     * Returns true if any of the values specified in the source array are contained
+     * in the <name>Iterable, and false if they are not.
+     *
+     * @since 11.0
+     */
+    default boolean containsAny(<type>... source)
+    {
+        if (this.size() \<= 32 || source.length \<= 32)
```

Comment:
can be simplified to:
```java
<name>Iterable inside = this;
if (this.size() \> 32 && source.length \> 32 && !this instanceof <name>Set)
{
    inside = this.toSet();
}
for (<type> item : source)
{
        if (inside.contains(item))
        {
            return true;
        }
}
return false;
```
Other 3 methods can be simplified similarly.

Reply:
Good point! I simplified all 4 methods and updated the PR.